### ISSUE TITLE
Price Forecast

### ISSUE THREAD

**Issue by kineticscreen** [opennem/opennem#66]
Is there any plan to include the forward spot price forecasts in the dataset?

**Comment by nc9** [comment]
Yes there are! I'll set this issue for the next release and you can track it here - this has been on the backburner for a while now and is overdue.

**Comment by kineticscreen** [comment]
That's great! I've been relying on Amber Electric's API which they're deprecating in favour of a private one just for customers (which I'm not) so that will be great!

**Comment by nc9** [comment]
I wasn't aware they're deprecating their public interface - thanks for letting me know. If I can get this out soon I'll tweet out that we've added support. I'll check if there's anything else we could add to make it compat with what they're removing.

**Comment by kineticscreen** [comment]
I don't know if they've killed it or not, but it's died for in the latest version of HA.
I get in my logs:
`The amberelectric platform for the sensor integration does not support platform setup. Please remove it from your config.`

**Comment by kineticscreen** [comment]
@nc9 I'm not good at Github, so couldn't work out of there is a better way of doing this, but thought you might be able to help out here: https://github.com/bacco007/sensor.opennem/issues/16

**Comment by nerfherder** [comment]
Does the above commit close this issue? I’m keen to bring this into https://github.com/bacco007/sensor.opennem if it is working

**Comment by purcell-lab** [comment]
I'm eager to access the price forecasts, is this usable now?

Would it be possible to provide an example of how to access the price forecast?

**Comment by nc9** [comment]
merging this into #500 - it's coming
